feature-libs/asm/root/config/asm-config.ts

```diff
@@ -3,7 +3,9 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-import { OccConfig } from '@spartacus/core';
+import { OccConfig, User } from '@spartacus/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { CustomerListColumnActionType } from '../model/customer-list.model';
 
 export abstract class AsmConfig extends OccConfig {
```

Comment:
no need to `extends OccConfig`, right? This config has nothing related to occ.

Reply:
I suppose so, if it is old. `CmsConfig` and `AnonymousConsentsConfig` extend `OccConfig`, unless they're not meant to.

Reply:
extends OccConfig removed.. thanks